Wireframe/css/style.css

```diff
@@ -0,0 +1,128 @@
+:root{
+    --background-color: rgb(43, 42, 42);
+    --text-color: white;
+    --link-color: rgb(64, 126, 226);
+    --article_background:rgb(87, 85, 85);
+
+    
+
+}
+html{
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+}
+body{
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 5px 0 5px 5px;
+
+    background-color: var(--background-color);
+    color: var(--text-color);
+    font-family: 'Courier New', Courier, monospace;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-shadow: 1px 0px black;
+
+}
+ .header-c{
+    width: 60%;
+    text-align: left;
+    justify-content: left;
+    margin-bottom: 20px;
+    padding: 5px;
+    
+    
+ }
+ .header-c h2{
+    text-align: center;
+    font-size: 30px;
+ }
+
+ .section{
+    
+    padding: 20px;
+    width: 75%;
+    
+    background-color: var(--article_background);
+    display: grid;
+    grid-template-columns: 1fr;
```

Comment:
Do you need this rule? What happens when you remove it? Try unchecking it in Devtools to test your prediction.